src/lib/containers/markdown/MarkdownEditor.tsx

```diff
@@ -107,6 +124,15 @@ export const MarkdownEditor: React.FunctionComponent<MarkdownEditorProps> = ({
 
   return (
     <div className="bootstrap-4-backport MarkdownEditor">
+      {forumId && (
+        <input
+          placeholder="Title"
+          type="text"
+          style={{ width: '100%', padding: '4px' }}
+          value={title}
+          onChange={e => setTitle(e.target.value)}
+        />
+      )}
```

Comment:
This should be in a `ForumThreadEditor` component, and rendered next to a `MarkdownEditor`, instead of inside of it, since the title field is forum-thread specific.